spec/forms/password_form_spec.rb

```diff
@@ -4,10 +4,6 @@
   subject { PasswordForm.new(build_stubbed(:user)) }
 
   it do
-    is_expected.to validate_confirmation_of(:password)
-  end
```

Comment:
This change was necessary in order to test the new password-related event tracking. I wanted the test to be as efficient as possible, and without this change, I would have had to create several new unnecessary stubs, making the test more complicated.